app/Models/Test.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Test extends Model
+{
+    //
```

Comment:
I would expect a method that implements the BelongsTo relations hip to its owning course